### DISCUSSION TITLE
NIST Douglas Fir simulation bulk densities

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13596] [Q&A] [answered]
Dear developers,

I was hoping someone could shed some light on the used bulk densities in the NIST Douglas Fir validation simulations. If I sum the dry bulk densities of the four particle classes, I obtain for the 2-meter tree with M=14% and M=49% bulk densities of 2.57 and 4.11 kg/m^3, respectively. The total dry bulk densities reported by Mell et al. for their Case 1 and Case 2 experiments were 2.64 and 5.86 kg/m^3, respectively. For Case 1, the difference is relatively small, however the Case 2 bulk density differs significantly from the reported value by Mell et al. I was wondering what might be the reason for this.

**Comment by ericvmueller** [comment]
I believe that number is the total mass in the crown, not the bulk density. In Table 3 of the paper the bulk density is 3.95 kg/m3 for the 49% MC tree. This number is much closer to the FDS value. As for the slight discrepancy in both cases, I'm not sure.

**Reply by r-broek** [reply, marked as answer]
It seems I misread that, thank you for the correction

**Comment by mcgratta** [comment]
The discrepancy might just be due to some miscalculation I made in setting these cases up. Adjust the values to what they should be for consistency.

**Comment by ericvmueller** [comment]
@ruddymell any thoughts here - something we are overlooking or should these values be adjusted?

**Comment by ericvmueller** [comment]
I made some very minor tweaks to the values to avoid further confusion: #13635
